workflow/rocoto/gfs_tasks.py

```diff
@@ -868,6 +868,18 @@ def gempak(self):
 
         return task
 
+    def npoess(self):
+
+        deps = []
+        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}post'}
```

Comment:
Looking at the ecflow trigger [here](https://github.com/NOAA-EMC/global-workflow/blob/62d8e99e77ffe44784f4c61c3f2296c6d1a72413/ecf/defs/gfs_00.def#L2095), this job depends on `jgfs_atmos_post_anl` and not the metatask for post-processing forecasts.
I think the dependency here is `postanl`

Reply:
Thank you. Another result of cut-and-paste and not knowing what these jobs do (other than satellite related post-processing).